web/src/engine/websites/Kakaopage.ts

```diff
@@ -0,0 +1,220 @@
+                sortType: $sortType
+              ) {
+                totalCount
+                pageInfo {
+                  hasNextPage
+                  endCursor
+                  hasPreviousPage
+                  startCursor
+                }
+                edges {
+                  cursor
+                  node {
+                    ...SingleListViewItem
+                  }
+                }
+              }
+            }
+            fragment SingleListViewItem on SingleListViewItem {
+              id
+              type
+              single {
+                productId
+                id
+                isFree
+                thumbnail
+                title
+                slideType
+              }
+            }
+
+        `;
+        const vars = {
+            seriesId: parseInt(manga.Identifier),
+            boughtOnly: false,
+            sortType: 'asc',
+            after: nextCursor
+        };
+        const request = new FetchRequest(new URL('/graphql', this.URI).href, {
+            headers: {
+                Referer: this.URI.href,
+                Origin: this.URI.href
+            }
+        });
+        const operationName = 'contentHomeProductList';
+        return await FetchGraphQL<APIChapters>(request, operationName, gql, JSON.stringify(vars));
+    }
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+        const gql = `
+            query viewerInfo($seriesId: Long!, $productId: Long!) {
+              viewerInfo(seriesId: $seriesId, productId: $productId) {
+                viewerData {
+                  ...ImageViewerData
+                  __typename
+                }
+                __typename
+              }
+            }
+
+            fragment ImageViewerData on ImageViewerData {
+              type
+              imageDownloadData {
+                ...ImageDownloadData
+                __typename
+              }
+            }
+
+            fragment ImageDownloadData on ImageDownloadData {
+              files {
+                ...ImageDownloadFile
+                __typename
+              }
+            }
+
+            fragment ImageDownloadFile on ImageDownloadFile {
+              no
+              secureUrl
+            }
+
+        `;
+        const vars = {
+            productId: parseInt(chapter.Identifier),
+            seriesId: parseInt(chapter.Parent.Identifier)
+        };
+        const request = new FetchRequest(new URL('/graphql', this.URI).href, {
+            headers: {
+                Referer: this.URI.href,
+                Origin: this.URI.href
+            }
+        });
+        const operationName = 'viewerInfo';
+
+        try {
+            const data = await FetchGraphQL<APiPages>(request, operationName, gql, JSON.stringify(vars));
+            return data.viewerInfo.viewerData.imageDownloadData.files.map(page => new Page(this, chapter, new URL(page.secureUrl)));
+        } catch (error) {
+            return [];
```

Comment:
Is ignoring this error by intention?
May be better to just let the caller handle the error.